assets/scss/templates/_guidelines.scss

```diff
@@ -233,3 +233,467 @@ body {
   justify-content: center;
   align-items: center;
 }
+
+
+/*-----------------------------------------------------------------------------------------------*/
+/* New Layout */
+
+.wrapper-main {
```

Comment:
What do you think about using a prefix to clarify that these styles are internal only?

Reply:
Yes, sure. If you like it I will make the comments so that it remains in SassDoc there we can clarify that it is for private / internal use

Reply:
Good idea, although maybe we could choose to use these styles outside the library. 

A clear naming and documentation would help.

Reply:
I can use the prefix we currently use for the `dp-component` components

Reply:
I agree, it would be great to differentiate this internal menu from the resto of the library.

Reply:
i agree too!